/limo_control:
  ros__parameters:
    default_speed: 0.3
    p_gain: 0.006
    slow_down_ratio: 0.6
    use_sim_time: false

